src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -750,8 +750,10 @@ private void OnUpdateGraphCompleted(AsyncTask task)
                 var node = workspace.Nodes.FirstOrDefault(n => n.GUID == guid);
                 if (node == null)
                     continue;
-
-                node.Warning(message.Value); // Update node warning message.
+                using (node.PropertyChangeManager.SetPropsToSuppress(nameof(NodeModel.ToolTipText), nameof(NodeModel.State)))
```

Comment:
I did not know you can declare an array by simply specifying a comma-separated list of values.

Reply:
Tibi suggested using the params modifier which allows this.